core/services/ocrcommon/telemetry.go

```diff
@@ -365,41 +365,41 @@ func (e *EnhancedTelemetryService[T]) getPricesFromResults(startTask pipeline.Ta
 		if benchmarkPriceTask.Result.Error != nil {
 			e.lggr.Warnw(fmt.Sprintf("got error for enhanced EA telemetry benchmark price, job %d, id %s: %s", e.job.ID, benchmarkPriceTask.Task.DotID(), benchmarkPriceTask.Result.Error), "err", benchmarkPriceTask.Result.Error)
 		} else {
-			benchmarkPrice, ok = benchmarkPriceTask.Result.Value.(float64)
-			if !ok {
-				e.lggr.Warnf("cannot parse enhanced EA telemetry benchmark price, job %d, id %s (expected float64, got type: %T)", e.job.ID, benchmarkPriceTask.Task.DotID(), benchmarkPriceTask.Result.Value)
+			benchmarkPrice, err = getResultFloat64(benchmarkPriceTask)
+			if err != nil {
+				e.lggr.Warnf(fmt.Sprintf("cannot parse enhanced EA telemetry benchmark price, job %d, id %s", e.job.ID, benchmarkPriceTask.Task.DotID()), "err", err)
 			}
 		}
 	}
 
 	bidTask := allTasks.GetNextTaskOf(*benchmarkPriceTask)
 	if bidTask == nil {
 		e.lggr.Warnf("cannot parse enhanced EA telemetry bid price, task is nil, job %d, id %s", e.job.ID)
-		return 0, 0, 0
+		return benchmarkPrice, 0, 0
 	}
 	if bidTask.Task.Type() == pipeline.TaskTypeJSONParse {
 		if bidTask.Result.Error != nil {
 			e.lggr.Warnw(fmt.Sprintf("got error for enhanced EA telemetry bid price, job %d, id %s: %s", e.job.ID, bidTask.Task.DotID(), bidTask.Result.Error), "err", bidTask.Result.Error)
 		} else {
-			bidPrice, ok = bidTask.Result.Value.(float64)
-			if !ok {
-				e.lggr.Warnf("cannot parse enhanced EA telemetry bid price, job %d, id %s (expected float64, got type: %T)", e.job.ID, bidTask.Task.DotID(), bidTask.Result.Value)
+			bidPrice, err = getResultFloat64(bidTask)
+			if err != nil {
+				e.lggr.Warnf(fmt.Sprintf("cannot parse enhanced EA telemetry bid price, job %d, id %s", e.job.ID, bidTask.Task.DotID()), "err", err)
```

Comment:
```suggestion
				e.lggr.Warnw(fmt.Sprintf("cannot parse enhanced EA telemetry bid price, job %d, id %s", e.job.ID, bidTask.Task.DotID()), "err", err)
```

Reply:
Fixed in [`eba6216` (#10828)](https://github.com/smartcontractkit/chainlink/pull/10828/commits/eba621657f7d96acf33f7ee9596150ba8ea4104d)